deploy-to-local.sh

```diff
@@ -85,4 +89,32 @@ echo "Moblink relay installed successfully."
 # shellcheck disable=SC2029
 ssh "$SSH_TARGET" "cd $BELAUI_PATH; bash ./override-belaui.sh"
 
+# Check if CeraUI should be installed
+if [ "$USE_CERAUI" = "true" ]; then
+  echo "Downloading and installing CeraUI content"
+
+  # Create a temporary script to download and extract CeraUI on the remote machine
+  TMP_SCRIPT=$(cat <<'EOF'
+#!/bin/bash
+set -e
+CERAUI_TEMP_DIR="$(mktemp -d)"
+cd "$CERAUI_TEMP_DIR"
+wget -q --show-progress CERAUI_RELEASE_URL
+tar xf CERAUI_RELEASE_TARBALL
+rsync -rltz --delete --chown=root:root "$CERAUI_TEMP_DIR/public/" BELAUI_PATH/public/
```

Comment:
Pretty sure the tarball does not have the public directory, right? Or do i get an old version?

```suggestion
rsync -rltz --delete --chown=root:root "$CERAUI_TEMP_DIR/" BELAUI_PATH/public/
```

Reply:
sorry for that, the first versions I was generating included the public folder path then I removed it